models/pwa.js

```diff
@@ -21,7 +21,8 @@ const Manifest = require('../models/manifest');
 
 class Pwa {
   constructor(manifestUrl, manifestModel) {
-    this.manifestUrl = manifestUrl;
+    // remove hash from url
+    manifestUrl && (this.manifestUrl = manifestUrl.replace(/#.*/, ''));
```

Comment:
You could reuse node's URL module for this:
```
removeHash(urlString) {
  const url = URL.parse(manifestUrl);
  url.hash = '';
  return url.format();
}
```
Not shorter than your solution though...